lib/app/views/dashboard.erb

```diff
@@ -15,13 +17,28 @@
           <% if current_user.ongoing.any? %>
             <p>Keep working on these problems, or mark them as done.</p>
             <ul class="nav nav-stacked" id="menu-current" role="menu">
-              <% current_user.ongoing.each do |submission| %>
+              <% current_user.ongoing[0..4].each do |submission| %>
```

Comment:
OK, I researched this a bit and learned about limits. Still kind of fuzzy on how it works and take vs limit.